server/views/pages/cookies/cookies-policy.njk

```diff
@@ -95,6 +95,38 @@
         <li>what you click on while you’re using the service</li>
       </ul>
 
+        <p class="govuk-body">Google Analytics sets the following cookies:</p>
+      {% from "govuk/components/table/macro.njk" import govukTable %}
```

Comment:
Generally better at the top of the page so you can see imports more easily.

Reply:
Moved to top